docs/content/interacting/consistency.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title:  Query Consistency Modes
+sidebar_position: 7
+slug: /interacting/consistency
+description: Query Consistency Modes
+---
+
+import {
+  DocumentationNotice,
+  IntroductionSection,
+  RelatedSection,
+  ProductName,
+  ProductNameFormat,
+} from '@components/Docs';
+
+# Query Consistency Modes
+
+<DocumentationNotice />
+
+:::info Note
+The Consistency parameter is an experimental feature in OpenFGA and needs to be enabled with the `enable-consistency-params` parameter, e.g. `openfga run --experimentals enable-consistency-params`.
+:::
+
+## Background
+
+When querying <ProductName format={ProductNameFormat.ShortForm}/> using any of the query APIs like [check, expand, read, list-objects and list-users requests](./relationship-queries.mdx), you can specify a query consistency parameter that can have one of the following values:
+
+| Name                        | Description                                                                                                   |  
+|-----------------------------|---------------------------------------------------------------------------------------------------------------|
+| MINIMIZE_LATENCY (default)  | <ProductName format={ProductNameFormat.ShortForm}/> will serve queries from the cache when possible           | 
+| HIGHER_CONSISTENCY          | <ProductName format={ProductNameFormat.ShortForm}/> will skip the cache and query the the database directly   |
+
+If you write a tuple and you immediately make a check on a relation affected by that tuple using `MINIMIZE_LATENCY`, the tuple change might not be taken in consideration if <ProductName format={ProductNameFormat.ShortForm}/> serves the result from the cache.
+
+## When to use higher consistency
+
+When specifying `HIGHER_CONSISTENCY` you are trading off consistency for latency. Always specifying `HIGHER_CONSISTENCY` will have a significant impact in performance.
+
+If you have a use case where higher consistency is needed, it's recommended that whenever possible, you decide in runtime the consistency level you need. If you are storing a timestamp indicating when a resource was last modified in your database, you can use that to decide the kind of request you do.
+
+For example, if you share `document:readme` with a `user:anne` and you update a `modified_date` field in the `document` table when that happens, you can write code like the below when calling `check("user:anne", "can_view", "document:readme")` to avoid paying the price of additional latency when calling the API.
+
+```javascript
+if (date_modified + cache_time_to_live_period > Date.now()) {
```

Comment:
Not sure if we should suggest this as this only works for direct assignments.

Reply:
Wouldn't you need to do something similar for Zookies?

Reply:
Yep! In a followup maybe we can expand on when and how to use it, but does not block merge